### PR TITLE
[Python API] Rename offline_transformations in the new api

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - Rename offline transformations from `offline_transformation_pybind`

### Tickets:
 - 69196
 - 75817

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

**Review by nosovmik — APPROVED** [review]
model-optimizer target is successfully built

**Review by ilyachur — APPROVED** [review]

File: tools/pot/openvino/tools/pot/graph/graph_utils.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from openvino.tools.mo.utils.logger import init_logger
 from openvino.runtime import Core  # pylint: disable=E0401,E0611
 from openvino.runtime.passes import Manager # pylint: disable=E0401,E0611
-from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+from openvino.offline_transformations import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
```
Comment (KodiaqQ) [inline tools/pot/openvino/tools/pot/graph/graph_utils.py:12]: Please add try-except block with the old and new import to avoid POT import errors.

Comment (akuporos) [reply tools/pot/openvino/tools/pot/graph/graph_utils.py]: done

**Comment by azhogov** [comment]
CI passed: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/770

File: tools/pot/openvino/tools/pot/graph/graph_utils.py [hunk 2]
```diff
@@ -9,7 +9,10 @@
 from openvino.tools.mo.utils.logger import init_logger
 from openvino.runtime import Core  # pylint: disable=E0401,E0611
 from openvino.runtime.passes import Manager # pylint: disable=E0401,E0611
-from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+try:
+    from openvino.offline_transformations import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+except ImportError:
+    from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
```
Comment (ilya-lavrenov) [inline tools/pot/openvino/tools/pot/graph/graph_utils.py:15]: why do we need it? where POT can take `offline_transformations_pybind `?

Comment (akuporos) [reply tools/pot/openvino/tools/pot/graph/graph_utils.py:15]: It was added to pass POT CI -- it's a separate action to switch their CI for using OpenVINO from this branch,
This addition was agreed with POT team (@KodiaqQ) and later will be removed by him.

**Pull request merged**